test/unix-connect-test.js

```diff
@@ -88,4 +88,37 @@ vows.describe('unix-connect').addBatch({
       transport.close();
     }
   }
+}).addBatch({
+  'Logging works if server comes up again': {
+    topic: function () {
+      var self = this;
+      var n = 2;
```

Comment:
At that time, the message in buffer is "data3", and the next message will be data 4, so at that time, it seemed convenient to set n to 2, to be synced with the "times" global variable (which represents the number added to the message).